questionnaire/2020/questions.yaml

```diff
@@ -285,6 +285,15 @@
     - 2 to 5 years 
     - More than 5 years
 
+  - title: Does your IT organization have a formal and/or sponsored initiatives in place to expand and improve the use of automation on the network?
```

Comment:
Remove the "a" in "Does your IT organization have a formal and/or sponsored initiatives", so everything is plural.